tests/tools.vitruv.applications.demo.insurancefamilies.tests/src/tools/vitruv/applications/demo/insurancefamilies/tests/util/InsuranceQueryUtil.xtend

```diff
@@ -0,0 +1,20 @@
+package tools.vitruv.applications.demo.insurancefamilies.tests.util
+
+import edu.kit.ipd.sdq.activextendannotations.Utility
+import tools.vitruv.framework.views.View
+
+import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.claimOne
+//import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.
+import edu.kit.ipd.sdq.metamodels.insurance.InsuranceDatabase
+
+@Utility
+class InsuranceQueryUtil {
+	
+	static def claimInsuranceDatabase(View view){
+		view.getRootObjects(InsuranceDatabase).claimOne
+	}
+	
+	static def claimInsuranceClient(InsuranceDatabase insuranceDatabase, String firstName, String lastName){
+		insuranceDatabase.insuranceclient.filter[c | c.name == firstName + " " + lastName].claimOne
```

Comment:
In Xtend you can often shorten these statements from
`c | c.name == firstName + " " + lastName`
to
`name == firstName + " " + lastName`